### PR TITLE
Remove signing for templates, now that we don't have them anymore.

### PR DESCRIPTION
**Pull request by livarcocc**
@333fred @dotnet/dotnet-cli

### CODE REVIEW

File: build/Signing/SignToolTemplateConfig.json [hunk 1]
```diff
@@ -1,14 +0,0 @@
-{
-    sign: [
-        {
-            "certificate": "VsixSHA2",
-            "strongName": null,
-            "values": [
-                "Microsoft.VisualStudio.ProjectSystem.CSharp.Templates.vsix",
-                "Microsoft.VisualStudio.ProjectSystem.CSharp.NetStandard.Templates.vsix",
```
Comment (nguerrera) [inline build/Signing/SignToolTemplateConfig.json]: One of these needs to stay because (for now) it still contains the actual SDK.

**Comment by 333fred** [comment]
How are the templates getting inserted now? Is something else going to create the signed template VSIXs?

File: build/Signing/SignToolTemplateConfig.json [hunk 2]
```diff
@@ -5,9 +5,6 @@
             "strongName": null,
             "values": [
                 "Microsoft.VisualStudio.ProjectSystem.CSharp.Templates.vsix",
```
Comment (jmarolf) [inline build/Signing/SignToolTemplateConfig.json]: This VSIX is no longer created.  `Microsoft.VisualStudio.Templates.CSharp.NetCore.vsmanproj` is the only thing that is still built and its only purpose is to package `Microsoft.NET.Sdk.swixproj`.

**Comment by jmarolf** [comment]
@333fred https://github.com/dotnet/project-system/pull/2053

**Review by 333fred — APPROVED** [review]

**Comment by livarcocc** [comment]
This passed on VSO.

**Pull request merged**